.github/workflows/pr-checks-clean-images.yaml

```diff
@@ -0,0 +1,27 @@
+name: "PR checks - Clean operator images"
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    types: [closed]
+
+env:
+  OPERATOR_REPO_REF: quay.io/redhat-developer/servicebinding-operator
+
+jobs:
+  clean-operator-images:
+    name: "Clean operator images (PR)"
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Git Repository
+        uses: actions/checkout@v2
```

Comment:
Do we really need to checkout the project since we do not refer any files later?

Reply:
We are refering the `./.github/actions/clean-images` action